lang/cs/Org.Apache.REEF.Client/API/IREEFClient.cs

```diff
@@ -37,11 +39,19 @@ public interface IREEFClient
 
         /// <summary>
         /// Submit the job described in jobSubmission to the cluster.
-        /// Expect IDriverHttpEndpoint returned after the call.
+        /// Expect IJobSubmissionResult returned after the call.
         /// </summary>
         /// <param name="jobSubmission"></param>
-        /// <returns>IDriverHttpEndpoint</returns>
+        /// <returns>IJobSubmissionResult</returns>
         [Unstable("0.13", "Working in progress for what to return after submit")]
-        IDriverHttpEndpoint SubmitAndGetDriverUrl(IJobSubmission jobSubmission);
+        IJobSubmissionResult SubmitAndGetJobStatus(IJobSubmission jobSubmission);
```

Comment:
Status right after submission is highly likely to be useless. What is the point of this API then?